app/lib/shared/configuration.dart

```diff
@@ -331,6 +249,9 @@ class Configuration {
 
 /// Configuration from the environment variables.
 class EnvConfig {
+  // Config Path points to configuratio file
```

Comment:
The constructor parameter is the last one, while this is the first field. Let's synchronize (e.g. move this to the last field).